### ISSUE TITLE
"Couldn't split downloaded image" (not due to RAM)

### ISSUE THREAD

**Issue by hyoretsu** [tachiyomiorg/tachiyomi#7457]
### Steps to reproduce

Try downloading a webtoon, in my latest case "The Tutorial is Too Hard" from MangaSee.

### Expected behavior

It shouldn't show that error, got it on some webtoons so far.

### Actual behavior

My reader's still laggy and I have an error message 24/7.

### Crash logs

[tachiyomi_crash_logs.txt](https://github.com/tachiyomiorg/tachiyomi/files/9041878/tachiyomi_crash_logs.txt)
[logcat_07-03-2022_01-01-37.txt](https://github.com/tachiyomiorg/tachiyomi/files/9041879/logcat_07-03-2022_01-01-37.txt)

### Tachiyomi version

r4553

### Android version

Android 10

### Device

Moto G7 Power

### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by hyoretsu** [comment]
Tl;dr Tachiyomi somehow doesn't seem to have permission (EACCESS) to create the splitted files on my folder/microSD, even though it has to save downloaded ones.
